src/Keboola/DbWriter/Writer/Redshift.php

```diff
@@ -253,13 +259,13 @@ private function execQuery($query)
         return null;
     }
 
-    private function getErrors()
+    private function getErrors(): array
     {
-        $query = $this->db->query("SELECT * FROM stl_load_errors WHERE query = pg_last_query_id();");
+        $query = $this->db->query('SELECT * FROM stl_load_errors WHERE query = pg_last_query_id()-1;');
```

Comment:
proc `-1` ?

Reply:
protože mi `pg_last_query_id()` vracelo současné query_id a ne toho který chci kontrolovat (proto -1)

Reply:
fakt?  to takhle neni v dokumentace https://docs.aws.amazon.com/redshift/latest/dg/PG_LAST_QUERY_ID.html

Reply:
po neúspěšném query je v tabulce:
![image](https://user-images.githubusercontent.com/12143866/67843786-f32c9a80-fafc-11e9-80ed-bbf1517020f0.png)
kterou dtb zavolá automaticky (není z naší strany) tak proto jsem dal `-1`

Reply:
ok, but I think this should have a test

Reply:
test by na to měl být tento:
https://github.com/keboola/db-writer-redshift/blob/ondra-require-standard-packages/tests/Keboola/FunctionalTest.php#L40

Reply:
@ondrajodas hmmm, to si mi furt nelibi.  a ten test failuje kdyz tam neni `-1`?

Reply:
@pivnicek jj když tam neni -1 tak failuje, proto jsem tam tu -1 dal

Reply:
@ondrajodas tak nechapu jak ten test nefailoval predtim?

Reply:
zkusím pustit build na masteru, který běžel naposledy před rokem 👍

Reply:
@pivnicek už failuje - https://travis-ci.org/keboola/db-writer-redshift/builds/394976544?utm_medium=notification

museli něco v redshiftu změnit

Reply:
huh, tak ok then.  jdem dopredu.  diky za trvpilivost ;-)